CMakeLists.txt

```diff
@@ -94,6 +97,10 @@ endif ()
 
 include(CheckCXXCompilerFlag)
 
+if(CMAKE_HOST_SYSTEM_NAME STREQUAL "Windows" AND CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
+    set(MSYS2 True)
```

Comment:
Could you rename this to something more generic? MSYS2 is, AFAIK, only one of several Windows GNU C toolchains. I for example use MinGW-w64 directly without using MSYS2 to compile the tiles build.

Reply:
I compile with CMake and set the compiler/linker executables through its command line. Some years ago I used `make` but I don't remember using `MSYS2=1` either.

Reply:
44448f4 should do it.